manual/installation.tex

```diff
@@ -686,81 +686,82 @@ \subsection{Installing on NERSC Edison Cray XC30}
-mkdir build_cori
-cd build_cori
-cmake -DQMC_EXTRA_LIBS=/usr/common/software/liblzma/20160630/hsw/lib/liblzma.a ..
-make -j 8
+mkdir build_cori_hsw
+cd build_cori_hsw
+cmake ..
+make -j 16
 ls -l bin/qmcpack
 \end{verbatim}
 
-When the above was tested on 15 July 2016, the following module and
+When the above was tested on 29 August 2017, the following module and
 software versions were present:
 
 \begin{verbatim}
 qmcpack@cori05:trunk> module list
 Currently Loaded Modulefiles:
-  1) nsg/1.2.0                             15) dvs/2.5_0.9.0-1.0502.2188.1.116.ari
-  2) modules/3.2.10.3                      16) alps/5.2.4-2.0502.9774.31.11.ari
-  3) eswrap/1.1.0-1.020200.1231.0          17) rca/1.0.0-2.0502.60530.1.62.ari
-  4) switch/1.0-1.0502.60522.1.61.ari      18) atp/1.8.3
-  5) intel/16.0.0.109                      19) PrgEnv-intel/5.2.82
-  6) craype-network-aries                  20) craype-haswell
-  7) craype/2.4.2                          21) cray-shmem/7.2.5
-  8) cray-libsci/13.2.0                    22) cray-mpich/7.2.5
-  9) udreg/2.3.2-1.0502.10518.2.17.ari     23) slurm/cori
- 10) ugni/6.0-1.0502.10863.8.29.ari        24) cray-hdf5/1.8.14
- 11) pmi/5.0.9-1.0000.10911.0.0.ari        25) gcc/5.1.0
- 12) dmapp/7.0.1-1.0502.11080.8.76.ari     26) cmake/3.3.2
- 13) gni-headers/4.0-1.0502.10859.7.8.ari  27) fftw/3.3.4.5
- 14) xpmem/0.1-2.0502.64982.5.3.ari        28) boost/1.59
+  1) modules/3.2.10.6                              14) alps/6.4.1-6.0.4.0_7.2__g86d0f3d.ari
+  2) nsg/1.2.0                                     15) rca/2.2.11-6.0.4.0_13.2__g84de67a.ari
+  3) intel/17.0.2.174                              16) atp/2.1.1
+  4) craype-network-aries                          17) PrgEnv-intel/6.0.4
+  5) craype/2.5.12                                 18) craype-haswell
+  6) udreg/2.3.2-6.0.4.0_12.2__g2f9c3ee.ari        19) cray-shmem/7.6.0
+  7) ugni/6.0.14-6.0.4.0_14.1__ge7db4a2.ari        20) cray-mpich/7.6.0
+  8) pmi/5.0.12                                    21) altd/2.0
+  9) dmapp/7.1.1-6.0.4.0_46.2__gb8abda2.ari        22) darshan/3.1.4
+ 10) gni-headers/5.0.11-6.0.4.0_7.2__g7136988.ari  23) cray-hdf5/1.10.0.3
+ 11) xpmem/2.2.2-6.0.4.0_3.1__g43b0535.ari         24) cmake/3.3.2
+ 12) job/2.2.2-6.0.4.0_8.2__g3c644b5.ari           25) fftw/2.1.5.9
+ 13) dvs/2.7_2.2.31-6.0.4.1_6.1__gb3b87e6          26) boost/1.61
 \end{verbatim}
 
 \subsection{Installing on NERSC Cori, Xeon Phi KNL Knight's Landing partition, Cray XC40}
 The second phase of NERSC's Cori uses Intel
-Xeon Phi Knight's Landing (KNL) nodes. Substantive optimizations for QMCPACK on KNL will
-be forthcoming. The following build recipe ensures that the code
+Xeon Phi Knight's Landing (KNL) nodes. The following build recipe ensures that the code
 generation is appropriate for the KNL nodes:
 
 \begin{verbatim}
+module unload cray-libsci  
+module load boost  
 module load cray-hdf5
 module load cmake
 module load fftw
 export FFTW_HOME=$FFTW_DIR/..
```

Comment:
Similar thing for PHDF5 and FFTW

Reply:
won't check phdf5 in this PR. No time.

Reply:
Turned out to be simple. Please merge if you are OK with the updated PR.